LoginAndSignUp/login-signup.css

```diff
@@ -0,0 +1,53 @@
+
+
+.login-signup-container{
+    background-color: var(--color-white);
+    min-height: 45rem;
+    min-width: 40rem;
+}
+
+.page-content{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.login-signup-container{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
```

Comment:
Maybe make separate utility class for these styles that will be helpful throughout the project.
```css
    display: flex;
    justify-content: center;
    align-items: center;
```